markdown.config.js

```diff
@@ -27,8 +29,8 @@ const buildRulesTable = (rows) => {
     .join('\n');
 };
 
-const BASIC_RULES = () => buildRulesTable(ruleTableRows.filter((rule) => !rule.includes('react/jsx-')));
-const JSX_RULES = () => buildRulesTable(ruleTableRows.filter((rule) => rule.includes('react/jsx-')));
+const BASIC_RULES = () => buildRulesTable(ruleTableRows.filter((rule) => !arrayIncludes(rule, 'react/jsx-')));
+const JSX_RULES = () => buildRulesTable(ruleTableRows.filter((rule) => arrayIncludes(rule, 'react/jsx-')));
```

Comment:
This is string includes which is available as of Node 4 so we should be able to use it.

Reply:
oops, too-aggressive grep :-)